.eslintignore

```diff
@@ -1,6 +1,6 @@
 !/.storybook
 /.storybook/data/*
-/dist/*
+**/dist/*
```

Comment:
Same here, this change is no longer needed, right?

Reply:
Correct, I'll remove it. 👍🏻